CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -0,0 +1,58 @@
+- Add route info in the output of `antctl get bgproutes`. ([#6803](https://github.com/antrea-io/antrea/pull/6803) [#6835](https://github.com/antrea-io/antrea/pull/6835), [@Atish-iaf])
+
+### Changed
+
+- Promote feature `EgressSeparateSubnet` from Alpha to Beta. ([#6982](https://github.com/antrea-io/antrea/pull/6982), [@luolanzone])
+- Promote feature `ServiceExternalIP` from Alpha to Beta. ([#6903](https://github.com/antrea-io/antrea/pull/6903), [@xliuxu])
+- Accept configurations with no collector / sink to avoid crash issue for Flow Aggregator. ([#7006](https://github.com/antrea-io/antrea/pull/7006), [@antoninbas])
+- More efficient IP check to query `NodeRouteController` for storing all the PodCIDRs to determine whether the source / destination IPs are Pod IPs (NodeIPAM only) for `FlowExporter`. ([#6960](https://github.com/antrea-io/antrea/pull/6960), [@antoninbas])
+- Remove stale OVS interfaces in the CNIServer reconciler if the original Pod interface is disconnected. ([#6919](https://github.com/antrea-io/antrea/pull/6919), [@wenyingd])
+- Remove local ASN range limitation in `BGPPolicy` API. ([#6914](https://github.com/antrea-io/antrea/pull/6914), [@hongliangl])
+- Support providing a fixed public host key for SFTP uploads with a new field `hostPublicKey` to `PacketCapture` and `SupportBundleCollection` CRDs. ([#6848](https://github.com/antrea-io/antrea/pull/6848), [@antoninbas])
+- Skip processing Service without IPPool annotation `service.antrea.io/external-ip-pool` for `ServiceExternalIP` feature. ([#6950](https://github.com/antrea-io/antrea/pull/6950), [@xliuxu])
+- Improve the document of the configuration option `egress.exceptCIDRs`. ([#6828](https://github.com/antrea-io/antrea/pull/6828), [@tnqn])
+- Upgrade CNI plugins from v1.5.1 to v1.6.2. ([#6796](https://github.com/antrea-io/antrea/pull/6796), [@luolanzone])
+- Push Antrea Ubuntu-based images to `ghcr.io`. ([#6834](https://github.com/antrea-io/antrea/pull/6834), [@antoninbas])
+- Upgrade go-ipfix to 0.13.0. ([#6998](https://github.com/antrea-io/antrea/pull/6998), [@antoninbas])
+
+### Fixed
+
+- Add `-ComputerName localhost` explicitly for VMSwitch commands to avoid potential validation issue on Windows with Active Directory. ([#6985](https://github.com/antrea-io/antrea/pull/6985), [@XinShuYang])
+- Fix that Antrea L7NetworkPolicies do not handle Service traffic correctly. ([#6941](https://github.com/antrea-io/antrea/pull/6941), [@hongliangl])
+- Disable TX checksum offload for Antrea host gateway interface when `disableTXChecksumOffload` is set to `true`. ([#6843](https://github.com/antrea-io/antrea/pull/6843), [@hongliangl])
+- Add `fqdnCacheMinTTL` configuration for Antrea ClusterNetworkPolicy to ensure that resolved IPs are included in data path rules as long as the application caches them. ([#6808](https://github.com/antrea-io/antrea/pull/6808), [@hkiiita])
```

Comment:
This is not specific to ClusterNetworkPolicy, s/Antrea ClusterNetworkPolicy/Antrea-native policies/